pages/careers/_open-positions.html

```diff
@@ -9,15 +9,20 @@ <h2 class="bold">Open Positions</h2>
       looking for:
     </p>
 
-    <h3>Software Engineering</h3>
+    <h3>Product</h3>
     <ul>
       <li><a href="#principal-product-manager">Principal Product Manager (US time zones)</a></li>
+    </ul>
+    <h3>Software Engineering</h3>
+    <ul>
       <li><a href="#principal-software-engineer-iac">Principal Software Engineer - IaC Library &amp; Platform (US time zones)</a></li>
       <li><a href="#senior-software-engineer-iac">Senior Software Engineer - IaC Library &amp; Platform (US time zones)</a></li>
+      <li><a href="#software-engineer-contractor">Software Engineering Contractor (US and EU time zones) </a></li>
```

Comment:
This href isn't working, I'd also swap out `US and EU time zones` for `US or EU time zones`